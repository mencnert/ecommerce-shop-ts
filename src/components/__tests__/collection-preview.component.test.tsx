import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import CollectionPreview from "../collection-preview/collection-preview.component";

it("should match snapshot", () => {
  const data = {
    title: "title",
    items: [
      {
        id: 1,
        name: "name",
        price: 20,
        imageUrl: "url",
      },
    ],
  };
  const wrapper = shallow(<CollectionPreview {...data} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
