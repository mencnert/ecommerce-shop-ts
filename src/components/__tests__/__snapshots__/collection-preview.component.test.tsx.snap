// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="collection-preview"
>
  <h1
    className="title"
  >
    TITLE
  </h1>
  <div
    className="preview"
  >
    <CollectionItem
      id={1}
      imageUrl="url"
      key="1"
      name="name"
      price={20}
    />
  </div>
</div>
`;
